modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -1,3 +1,63 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+Subscriptions are a core concept of GraphQL and one which allow us to enable real time updates and synchronisation of data.
+GraphQL Subscriptions use web sockets to enable clients to subscribe to published changes. {sync-client} comes with websocket and therefore
+subscription support out of the box. To enable subscriptions on your client set the following paramater in the {sync-client} config object.
+A DataSyncConfig interface is also available from {sync-client} if you wish to use it.
+
+An example of configuring the client for subscriptions can be seen below.
+
+```
+const config: DataSyncConfig {
+    ...
+    wsUrl: { WEB_SOCKET_URL_HERE }
+    ...
+}
+```
+Where wsUrl is the full URL of the websocket endpoint of your GraphQL server. Once this object is created, use it to initialise {sync-client} with:
+
+```
+import { createClient } from '@aerogear/voyager-client';
+
+const client = createClient(config)
+```
+
+Once the client is configured we can then make use of subscriptions as follows.
+
+```
+
+getTasks() {
+  const tasks = client.watchQuery<Tasks>({
+      query: GET_TASKS,
+      fetchPolicy: 'cache-first',
+      errorPolicy: 'none'
+  });
+
+  tasks.subscribeToMore({
+        document: TASK_MUTATED_SUBSCRIPTION,
+        updateQuery: (prev, { subscriptionData }) => {
```

Comment:
This is specific aproach we use inside our app. It will require explanation before showing code.
We can leave update query empty and have chapter on this?